enterprise/backup/src/test/java/org/neo4j/backup/OnlineBackupCommandIT.java

```diff
@@ -142,6 +158,56 @@ public void makeSureBackupCanBePerformedWithCustomPort() throws Exception
         assertEquals( getDbRepresentation(), getBackupDbRepresentation( "customport" ) );
     }
 
+    @Test
+    public void onlyTheLatestTransactionIsKeptAfterIncrementalBackup() throws Exception
+    {
+        // given database exists with data
+        int port =  4446;
+        startDb( "" + port );
```

Comment:
be careful with this hardcoded port in more recent versions

Reply:
Yes, merge resolution changes it to use PortAuthority in 3.3